frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -129,6 +128,14 @@ action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-fail=Sli
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.of-current=Branch <b>{0}</b> has been slid out but not deleted as it is currently checked out
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.with-delete=Branch <b>{0}</b> has been slid out and deleted
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.without-delete=Branch <b>{0}</b> has been slid out but not deleted
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.title=Git Machete Branch Deletion Suggestion
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.ok-text=_Delete local branches
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.no-text=_Keep local branches
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.message=\
+  <html>\
+    Git Machete is able to delete the local branches after sliding them out.<br> \
+    This is achieved by setting the git config <b>machete.slideOut.deleteLocalBranch</b> under the hood.\
```

Comment:
I'd formulate it in more user-oriented manner, like `you can change this setting by ...`

Reply:
Also, setting the config... to what value? ;)